src/main/kotlin/no/nav/syfo/domain/Foresporsel.kt

```diff
@@ -1,14 +1,55 @@
 package no.nav.syfo.domain
 
+import no.nav.syfo.util.nowUTC
 import java.time.OffsetDateTime
 import java.util.UUID
 
 @ConsistentCopyVisibility
-data class Foresporsel(
+data class Foresporsel private constructor(
     val uuid: UUID,
     val createdAt: OffsetDateTime,
     val arbeidstakerPersonident: Personident,
     val veilederident: Veilederident,
     val virksomhetsnummer: Virksomhetsnummer,
     val narmestelederPersonident: Personident,
-)
+) {
+    constructor(
+        arbeidstakerPersonident: Personident,
+        veilederident: Veilederident,
+        virksomhetsnummer: Virksomhetsnummer,
+        narmestelederPersonident: Personident,
+    ) : this(
+        uuid = UUID.randomUUID(),
+        createdAt = nowUTC(),
+        arbeidstakerPersonident = arbeidstakerPersonident,
+        veilederident = veilederident,
+        virksomhetsnummer = virksomhetsnummer,
+        narmestelederPersonident = narmestelederPersonident,
+    )
+
+    fun copyForTest(
```

Comment:
Fant ikke noen bedre måte å gjøre dette på..

(dvs med private constructor og ConsistentCopyVisibility=

Reply:
Kunne man heller endret på `generateForsporsel()` til å være `generateForsporsel(personident: Personident)`, så kan man heller bruke den i testene for å lage forespørsler i stedet for å bruke copy?